pkgs/mkpkgs

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# NOTE: you'll need to create a arm64 buildx container first
+# `docker buildx create --platform linux/arm64 --use`
+# `docker buildx inspect <name of container> --bootstrap`
+# `docker buildx ls` should show it running & ready for arm64
+
+set -eux
+
+ARCH=${ARCH:-linux/arm64,linux/amd64}
+POSTGRES_VER=${POSTGRES_VER:-16}
+
+# FIND THE ROOT GIT DIRECTORY
+DIR=$(git rev-parse --show-toplevel $(dirname $0)|head -n 1)
+
+# ITERATION FOR THE PACKAGES
+ITERATION=${ITERATION:-1}
+
+# GIT REVISION OF SOURCE
+REVISION=${REVISION:-"$(git rev-parse --short HEAD)"}
+
+# READ VERSIONS
+source "${DIR}/versions.txt"
+
+# BULID FOR REDHAT.
+for arch in $(echo $ARCH|tr ',' ' '); do
+    # NOTE: I have this building architectures sequentially because I'm on a
+    # laptop. On a beefy CI machine, it might be better to do this in one pass
+    # maxing the machine with parallel builds like the following example:
+    # `docker buildx build --platform linux/amd64,linux/arm64 --push`
+    ROCKY_IMAGE_NAME=omnigres:rocky-pgdg${POSTGRES_VER}-$(echo $arch|sed "s/linux\///")-$REVISION
+    ROCKY_CONTAINER_NAME=$(cat /proc/*/*/*/uuid||uuidgen) ;# WORKS ON LINUX & MACOS
```

Comment:
nitpick: may be use $arch, $POSTGRES_VER and other relevant details for the name? Easier to debug if things go wrong

Reply:
We can name things here however you like. It's not important to the output of the rpms.

Reply:
That's why it is a nitpick :)